lightly/transforms/phase_shift_transform.py

```diff
@@ -7,36 +7,49 @@
 
 
 class PhaseShiftTransform:
-    """Implementation of phase shifting transformation
+    """
+    Implementation of phase shifting transformation.
 
-    Adds a random phase shift `theta` (positive or negative),
-    to all components of the Fourier spectrum (`freq_image`) of the image and return it.
+    Applies a random phase shift `theta` (positive or negative) to the Fourier spectrum (`freq_image`) of the image and returns the transformed spectrum.
 
     Attributes:
-        dist:
-            Uniform distribution in `[p, q)` from which the magnitude of phase shift will be selected.
+        dist (torch.distributions.Uniform):
+            A uniform distribution in the range `[p, q)` from which the magnitude of the phase shift `theta` is selected.
+
+        include_negatives (bool):
+            A flag indicating whether negative values of `theta` should be included. If `True`, both positive and negative shifts are applied.
+
+        sign_dist (torch.distributions.Bernoulli):
+            A Bernoulli distribution used to decide the sign of `theta`, based on a given probability `sign_probability`, if negative values are included.
     """
 
-    def __init__(self, range: Tuple[float, float] = (0.4, 0.7)) -> None:
+    def __init__(
+        self,
+        range: Tuple[float, float] = (0.4, 0.7),
+        include_negatives: bool = False,

```

Comment:
Sure, we can include the negatives. While theta is sampled from a positive distribution, the equation does include both in the image I think they are just simplifying the approach. 
About removing `include_negatives` and `sign_probability`, I did consider that initially but I think that they are some nice to haves in case we want finer control over the transformation (moving away from the exact implementation of the paper), same as having `range` as parameter. If you think they are redundant then I do have the other version without them and I can push that.

Reply:
I'll check with the authors what the default setting is :)